core/src/main/java/tc/oc/pgm/match/MatchPlayerImpl.java

```diff
@@ -54,7 +51,7 @@
 import tc.oc.pgm.util.named.NameStyle;
 import tc.oc.pgm.util.nms.NMSHacks;
 
-public class MatchPlayerImpl implements MatchPlayer, Comparable<MatchPlayer> {
+public class MatchPlayerImpl implements MatchPlayer {
```

Comment:
This looks like a dangerous change

Reply:
The method was not used anywhere I could find, it was especially weird that it was scoped to `MatchPlayerImpl` and not `MatchPlayer`

Reply:
That's all fine until there's somewhere in the code that uses a TreeSet<MatchPlayer> or similar and you're doomed

Reply:
`Match` does not implement `Comparable`, do you think that is something to include back as well?

Reply:
It probably should for ease of use, but the example of something that could break doesn't necessarily need it to work